cypress/support/step-definitions/show-edit-pod-steps.js

```diff
@@ -96,9 +95,12 @@ Then('Show Edit Pod as value is set to {string}', (property) => {
     .should('have.class', `${SHOW_EDIT_POD_CONTENT}${property}`);
 });
 
-Then('Show Edit Pod buttonAlign value is set to {string}', (property) => {
-  showEditPodSecondaryWrapper()
-    .should('have.class', `${SHOW_EDIT_POD_WRAPPER}${property}`);
+Then('Show Edit Pod buttons are aligned left', () => {
+  showEditPodFooter().children().first().should('have.text', 'Save');
```

Comment:
if we are using this form like `showEditPodFooter().children()` only in those 2 steps, probably we should move the `children()` part to `index.js` file and after that use only `showEditPodFooter()`, what do you think about that?

Reply:
fixed